.gitignore

```diff
@@ -13,4 +13,6 @@ hachoir3.egg-info/
 .DS_Store
 
 # VSCode files
-.vscode/
\ No newline at end of file
+.vscode/
+
+env/
```

Comment:
Sorry, I'm new to pull requests. This is a folder to my virtualenv. Shouldn't made ir to pull request..